modules/install-osp-deploy-dualstack.adoc

```diff
@@ -164,4 +164,10 @@ ipv6.addr-gen-mode=0
 ----
 
 After you create and edit the file, reboot the installation host.
+====
+
+[NOTE]
+====
+The `ip=dhcp,dhcp6` kernel argument, which is set on all of the nodes, results in a Network Manager connection profile that is activated on multiple interfaces simultaneously.
+Because of this behavior, any additional network will have the same connection enforced with an identical UUID. If you need an interface-specific configuration, create a new connection profile for that interface so that the default connection is no longer enforced on it.
```

Comment:
Adding "single" per @mkowalski 

```suggestion
The `ip=dhcp,dhcp6` kernel argument, which is set on all of the nodes, results in a single Network Manager connection profile that is activated on multiple interfaces simultaneously.
Because of this behavior, any additional network will have the same connection enforced with an identical UUID. If you need an interface-specific configuration, create a new connection profile for that interface so that the default connection is no longer enforced on it.
```

I'm seeing NetworkManager and `NetworkManager` used in the docs. "Network Manager" doesn't work with any of our conventions, as I think about it, as the official name is "NetworkManager". Going with the non-backticked version for now, as the name truly seems to be that.